en/clinic-user-guide.md

```diff
@@ -0,0 +1,672 @@
+
+If the cluster cannot access the Internet, you can deploy Clinic Diag with the following step.
+
+1. Download the Clinic Diag chart.
+
+    If the cluster cannot access the Internet, it is unable to install Clinic Diag and other components by configuring the Helm repo. You need to download the chart files on a machine has Internet access and then copy them to the cluster.
+
+    To download Clinic Diag chart files, you can use the following command:
+
+    ```shell
+    wget http://charts.pingcap.org/diag-v0.7.1.tgz
+    ```
+
+    Copy `diag-v0.7.1.tgz` to the cluster and unpack it to the current directory.
+
+    ```shell
+    tar zxvf diag-v0.7.1.tgz
+    ```
+
+2. Download the Clinic Diag image.
+
+    You need to download the Clinic Diag image on a machine that has Internet access and then use the `docker load` command to load the images to the cluster.
+
+    The Clinic Diag image is `pingcap/diag:v0.7.1`. You can download and archive the image with the following command:
+
+    ```shell
+    docker pull pingcap/diag:v0.7.1
+    docker save -o diag-v0.7.1.tar pingcap/diag:v0.7.1
+    ```
+
+    Then, copy the archive to the cluster and use the `docker load` command to load the image to the cluster:
+
+    ```shell
+    docker load -i diag-v0.7.1.tar
+    ```
+
+3. Configure the `values-diag-collector.yaml` file.
+
+    Modify your access token in the `${HOME}/diag-collector/values-diag-collector.yaml` file.
+
+    Other configuration parameters such as `limits`, `requests`, and `volume` can be modified according to your needs.
+
+    > **Note:**
+    >
+    > To get the token, please refer to [Step 2: Log in to the Clinic Server and get an access token](#step-2-log-in-to-the-clinic-server-and-get-an-access-token)
+
+4. Install Clinic Diag.
+
+    Install Clinic Diag with the following command:
+
+    ```shell
+    helm install diag-collector ./diag --namespace=tidb-admin
+    ```
+
+    > **Note:**
+    >
+    > The `namespace` should be the same as TiDB Operator. If TiDB Operator is not deployed, please deploy TiDB Operator first and then deploy Diag.
+
+5. (Optional) Set the persistent volume.
+
+    This step can set the volume for Diag to persist its data. To set the volume, you can set `diag.volume` with the volume type in the `${HOME}/diag-collector/values-diag-collector.yaml` file. The following examples are PVC and Host:
+
+    ```
+    # Use PVC volume type
+    volume:
+      persistentVolumeClaim:
+        claimName: local-storage-diag
+    ```
+
+    ```
+    # Use Host volume type
+    volume:
+      hostPath:
+        path: /data/diag
+    ```
+
+    > **Note:**
+    >
+    > - It is not supported to set the volume on multiple disks.
+    > - Support any type of StorageClass.
+
+</div>
+<div label="Least privileges deployment">
+
+> **Note:**
+>
+> Least privileges deployment is to deploy Diag to the namespace of the cluster and Diag can only collect the data in the namespace but not across the namespaces.
```

Comment:
```suggestion
> Least privilege deployment is to deploy Diag to the namespace of the cluster so that Diag can collect data only in the namespace but not across namespaces.
```